cloud/google/update_log.sh

```diff
@@ -11,21 +11,30 @@ set -e
 GCLOUD="gcloud"
 
 Header "Updating log instances..."
-for i in `seq 0 $((${LOG_NUM_REPLICAS} - 1))`; do
+i=0
+while [ $i -lt ${MIRROR_NUM_REPLICAS} ]; do
   echo "Updating ${LOG_MACHINES[${i}]}"
   echo "${LOG_META[${i}]}" > /tmp/metadata.${i}
 
```

Comment:
Yeah, I figured that's better than just 'silently' leaving the cluster in an inconsistent state - at least this way the operator will have to do something about it.